source/js/Directions.js

```diff
@@ -7,55 +7,220 @@ class Directions extends HTMLElement {
     const shadow = this.attachShadow({ mode: "open" });
   }
 
-  set data(data) {
-    const style = `
-        .example {
+    set data(data) {
+        const style=`
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8460933)
```suggestion
    const style=`
```